Makefile

```diff
@@ -38,6 +58,9 @@ latest:
 	# (run by CI only for releases)
 	docker push $(REPO):latest
 
+check:
+	golangci-lint run --enable=unconvert,prealloc,bodyclose
```

Comment:
What do you think about taking the installation command from the workflows .yaml files, and instead add a make target for `make bin/golangci-lint` which runs the curl and install task.  This way one can run `make check` themselves without needing to have the linter pre-installed.

Reply:
The other option would be to go full-on workflows integration, letting the developer be in the dark about lint errors until they push (or set up and run the golangci-lint action locally themselves)
https://github.com/golangci/golangci-lint-action

This is a nice integration, but as a developer I would prefer the ability to run `make check` myself locally before submitting the PR.